samples/sample_capture.py

```diff
@@ -14,7 +14,14 @@ def _main():
         settings.processing.filters.outlier.removal.enabled = True
         settings.processing.filters.outlier.removal.threshold = 5.0
 
-        with camera.capture(settings) as frame:
+        settings.color = Settings2D()
+        settings.color.acquisitions.append(Settings2D.Acquisition())
+        settings.color.acquisitions[0].aperture = 5.6
+        settings.color.acquisitions[0].exposure_time = datetime.timedelta(
+            microseconds=8333
+        )
```

Comment:
Did you check that we dont have same bug here as we had in .NET where we didnt mutate the actual settings.color but instead a copy?

Reply:
Yeah. Added a commit to test this